### PR TITLE
📝 draft community/communication page

### PR DESCRIPTION
**Pull request by ebullient**
A draft with links to social accounts, etc.

Please improve wording. Should we include Discord or drop it? =)

### CODE REVIEW

File: COMMUNICATION.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
-# Communication Channels
+# Join Our Community
 
-TBD -- finalize all communication channels
+We rely on the contribution and participation of our members. Whether you're here to contribute code, share knowledge, or simply support the continued development of our open-source projects, our community spaces welcome everyone ready to make a positive impact.
 
-- Mailing lists are in `CONTACTS.yaml`
+## Engage, Learn, and Share
+
+- [**GitHub Discussions**](https://www.commonhaus.org/feed/index.rss): Share ideas, ask questions, and help shape the evolution of our collective effort.
```
Comment (aalmiray) [inline COMMUNICATION.md:7]: Shouldn't there be a direct link to the Discussions page (in order to post a topic) rather than just consuming the feed?

**Comment by aalmiray** [comment]
Is the Discord server supposed to be used just for CF stuff or also projects (those that decide to spin off a dedicated channel I suppose).

File: COMMUNICATION.md (hunk 1 above)
Comment (ebullient) [reply COMMUNICATION.md:7]: That's a bad link that is.

**Comment by ebullient** [comment]
> Is the Discord server supposed to be used just for CF stuff or also projects (those that decide to spin off a dedicated channel I suppose).

In my head: it's CF as a community (towards the IRC-feel that folks remember from Commonhaus .. mailing lists aren't really that for me anymore)

**Comment by ebullient** [comment]
> In that case it should be listed as well.

I think I will leave it as supporter-only benefit, and will add a link to the community on the supporters page to help curb spam.

I'll also double check with another community on how they do role assignment associated w/ support..

**Comment by ebullient** [comment]
Merged to make progress. We can make additional revisions as we go.

**Pull request merged**